src/input/pointer.js

```diff
@@ -560,7 +631,7 @@
      * // register on the 'pointerdown' event
      * me.input.registerPointerEvent('pointerdown', this, this.pointerDown.bind(this));
      */
-    obj.registerPointerEvent = function (eventType, rect, callback, floating) {
```

Comment:
Since it saves a reference to the rect, it check if the rect is floating in the first place.